cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/ResolveMavenDependenciesTask.java

```diff
@@ -51,6 +52,22 @@ public void execute(BuildContext buildContext) {
                 + "platform-libs";
         MavenResolver resolver = new MavenResolver(targetRepo);
 
+        if (manifest.getPlatform().getRepositories() != null && manifest.getPlatform().getRepositories().size() > 0) {
+            for (Repository repository : manifest.getPlatform().getRepositories()) {
+                String id = repository.getId();
+                String url = repository.getUrl();
+                String username = repository.getUser();
+                String password = repository.getToken();
+                if (id != null && url != null) {
+                    if (username != null && password != null) {
```

Comment:
If this step is done only if the `id` and `url` are not null, then the initialization of `username` and `password` variables can be done within the `if` condition.